amundsen_application/base/examples/example_mail_client.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import smtplib
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from http import HTTPStatus
+from typing import Dict, List
+
+from flask import Response, jsonify, make_response
+
+from amundsen_application.base.base_mail_client import BaseMailClient
+
+
+class MailClient(BaseMailClient):
+    def __init__(self, recipients: List[str]) -> None:
+        self.recipients = recipients
+
+    def send_email(self,
+                   subject: str,
+                   text: str,
+                   html: str,
+                   sender: str = None,
+                   recipients: List[str] = None,
+                   optional_data: Dict = None) -> Response:
+        if not sender:
+            sender = '< amundsen email >'  # change me
+        if not recipients:
+            recipients = self.recipients
+
+        # TODO refactor that, it's insecure!
+        sender_pass = '< amundsen email password >'  # change me
```

Comment:
nit: Even though you have a clear message here that this is to be changed when writing the actual client, maybe the example can encourage the use of getting this from an environment variable?

Reply:
Done